crystalfontz/dbus/client/__init__.py

```diff
@@ -37,3 +41,56 @@ async def staged_config(self: Self) -> StagedConfig:
             target_config=Config.from_file(active_config.file),
             active_config=active_config,
         )
+
+    async def effect_client(self: Self) -> DbusEffectClient:
```

Comment:
I don't actually like this API on the client. I think I'd rather remove this and push it into the CLI.